data_structures/linked_list/doubly_linked_list.c

```diff
@@ -0,0 +1,259 @@
+/**
+ * @file
+ * @brief   Doubly linked list, simple code.
```

Comment:
Provide a Wikipedia link in Markdown format.
If there's no Wikipedia link, provide another web source for algorithm explanation. 🙂

Reply:
Fixed up, thank you! :slightly_smiling_face:

Reply:
> Fixed up, thank you! 🙂

I dont think this suggested change has been implemented @Panquesito7
```suggestion
 * @brief Implementation of [Doubly linked list](https://en.wikipedia.org/wiki/Doubly_linked_list)
```